index.html

```diff
@@ -0,0 +1,141 @@
+        <section class="section_1">
+         <h1> Multi-Post <br> Stories</h1>
+         <ul>
+           <li class="list_1"> CANOPY </li>
+           <li class="list_2"><img src="images/Counter_1.png">Back End Dev</li>
+           <li class="list3"><img src="images/Counter_2.png">2015</li>
+               </ul>
+         <p>A daily selection of privately 
+           personalized reads; no accounts 
+           or sign-ups required.</p>   
+         <button class="btn_3"> html</button>
+         <button class="btn_4">css</button>
+         <button class="btn_5">javascript</button>
+         </section>
+         <button class="btn_2">See Project</button>
+     </div>
+
+     <div class="container">
+        <div> <img src="images/Portfolio_3.png" class="img_2"> </div>
+        <section class="section_1">
+         <h1> Tonic </h1>
+         <ul>
+           <li class="list_1"> CANOPY </li>
+           <li class="list_2"><img src="images/Counter_1.png">Back End Dev</li>
+           <li class="list3"><img src="images/Counter_2.png">2015</li>
+               </ul>
+         <p>A daily selection of privately
+            personalized reads; no accounts 
+            or sign-ups required.</p>   
+         <button class="btn_3"> html</button>
+         <button class="btn_4">css</button>
+         <button class="btn_5">javascript</button>
+         </section>
+         <button class="btn_2">See Project</button>
+     </div>
+
+     <div class="container">
+        <div> <img src="images/Portfolio_4.png" class="img_2"> </div>
+        <section class="section_1">
+         <h1> Multi-Post <br> Stories </h1>
+         <ul>
+           <li class="list_1"> CANOPY </li>
+           <li class="list_2"><img src="images/Counter_1.png">Back End Dev</li>
+           <li class="list3"><img src="images/Counter_2.png">2015</li>
+               </ul>
+         <p>A daily selection of privately
+            personalized reads; no accounts 
+            or sign-ups required.</p>   
+         <button class="btn_3"> html</button>
+         <button class="btn_4">css</button>
+         <button class="btn_5">javascript</button>
+        </section>
+        <button class="btn_2">See Project</button>
+     </div>
+
+     <div class="container_2">
+        <section class = "section_2">
+          <h2> About <br> Myself</h2>
+          <p> I’m a software developer! I can help you build a product,
+            feature or website Look through some of my
+            work and experience! If you like what you see and
+             have a project you need coded, don’t hestiate to contact me.
+          </p>
+          <a href="#" id="connect_1"> LET'S CONNECT</a>
+          <ul class="social_2">
+            <li><a href="https://github.com/david30-maker"><img src="images/github1.png" alt="github logo"></a></li>
+            <li><a href="https://www.linkedin.com/in/david-igbo-464196184"><img src="images/Linkedin_1.png" alt="in logo"></a></li>
```

Comment:
- [ ] Your project should look almost like the Figma design. Please ensure you implement this section exactly as the Figma design. Consider, the font type, font color, font size, and margin/padding. 
- [ ] The text LET'S CONNECT should be wrapped in either `h2` or `h3` because it serves as a header for the social media icons below it. Kindly fix this.
- [ ] Your Get My Resume button in this section does not have the same height and shape as the Figma design. Kindly fix this.

Figma design | Your work
---- | ----
![image](https://user-images.githubusercontent.com/84156714/213210221-39ddce6b-258e-4102-b6eb-00561571a48a.png) | ![image](https://user-images.githubusercontent.com/84156714/213210294-b055bca4-1a1c-4c67-aa75-04a760fb0b58.png)